{"name": "test_Combine[pyfixture4]", "status": "failed", "statusDetails": {"message": "AssertionError: assert '1019' == 1018", "trace": "pyfixture = ({'custid_child': 123, 'custid_parent': 11010, 'loginType': 1, 'merge_channel': 'weixin', ...}, (1018, 'custid %s already bind same type third'))\n\n    @pytest.mark.combine\n    @pytest.mark.flaky(reruns=1, reruns_delay=10)\n    def test_Combine(pyfixture):\n        # 请求\n        url = Combine.url\n        data = pyfixture[0]\n        res = request(url=url, data=data)\n        # print(data)\n>       assert res['return_code'] == pyfixture[1][0]\nE       AssertionError: assert '1019' == 1018\n\ntestcase\\Combineapi\\Combine_test.py:111: AssertionError"}, "parameters": [{"name": "pyfixture", "value": "({'custid_parent': 11010, 'custid_child': 123, 'sessionid_parent': 123123123, 'merge_channel': 'weixin', 'loginType': 1, 'result_format': 'json'}, (1018, 'custid %s already bind same type third'))"}], "start": 1611055388908, "stop": 1611055388939, "uuid": "e6db74bd-6d24-464c-9b0b-9a286f2abaeb", "historyId": "6d0671c5dd5b0c018c851e789ec446ef", "testCaseId": "539049fcbb1862a32a91c1f106a060f3", "fullName": "testcase.Combineapi.Combine_test#test_Combine", "labels": [{"name": "tag", "value": "combine"}, {"name": "tag", "value": "flaky(reruns=1, reruns_delay=10)"}, {"name": "parentSuite", "value": "testcase.Combineapi"}, {"name": "suite", "value": "Combine_test"}, {"name": "host", "value": "A06P-A90F12DAA"}, {"name": "thread", "value": "13252-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "testcase.Combineapi.Combine_test"}]}